packages/react-internal/src/Styling.ts

```diff
@@ -1,2 +1,4 @@
 import './version';
 export * from '@uifabric/styling';
+export * from '@fluentui/theme';
```

Comment:
Do you think it will be obvious enough to people that this is the top-level file where these things should be imported from? Or should they get separate top-level files (probably `Theme` and `ThemeProvider`)?

Reply:
Perhaps we should keep the Styling export separate from a Theme.tsx export, so that we can deprecate Styling?

Reply:
moved the new exports to `Theme.ts` (`lib/Theme`)